pkg/cmd/pulumi/login.go

```diff
@@ -145,6 +159,8 @@ func newLoginCmd() *cobra.Command {
 	}
 
 	cmd.PersistentFlags().StringVarP(&cloudURL, "cloud-url", "c", "", "A cloud URL to log in to")
+	cmd.PersistentFlags().StringVar(&defaultOrg, "org-name", "", "A default org to associate with the login. "+
+		"Please note, this is only available for Pulumi SaaS and Pulumi Self Hosted backends")
```

Comment:
We should probably use the same terminology as our documentation (https://www.pulumi.com/docs/intro/concepts/state/#backends):

```suggestion
		"Please note, this is only available in backends which support organizations, including the hosted and self-service Pulumi Service backends")
```